includes/blocks/course-theme/class-lesson-actions.php

```diff
@@ -81,9 +81,13 @@ private function render_take_quiz( string $quiz_permalink, bool $is_disabled ) :
 		$quiz_permalink = esc_url( $quiz_permalink );
 		$text           = esc_html__( 'Take quiz', 'sensei-lms' );
 
-		return '<a href="' . $quiz_permalink . '" class="sensei-course-theme__button is-primary" ' . $disabled . '>'
-			. $text .
-		'</a>';
+		return ( '
+			<form method="GET" action="' . $quiz_permalink . '">
+				<button type="submit" data-id="complete-lesson-button" class="sensei-course-theme__button is-primary" ' . $disabled . '>
```

Comment:
Changing the Take quiz button from to use a `<button>` tag instead of `<a>`, so it'd need less workaround around styling.